src/menu/menu.vue

```diff
@@ -1,12 +1,12 @@
 <template lang="jade">
-ul.mdl-menu.mdl-js-menu(v-bind:for.once='for')
+ul.mdl-menu.mdl-js-menu(v-bind:for.once='target')
```

Comment:
vue-2.0.0-rc.3 gave an error if you tried to name a prop 'for' because it is a reserved JavaScript word. I just tried reverting this to test again w/ rc4, and instead it seems to just completely fail to render, without any more details as to why.